Mathlib/Geometry/Manifold/IsManifold/Basic.lean

```diff
@@ -919,6 +919,10 @@ instance : AddCommGroup (TangentSpace I x) := inferInstanceAs (AddCommGroup E)
 instance : IsTopologicalAddGroup (TangentSpace I x) := inferInstanceAs (IsTopologicalAddGroup E)
 instance : Module 𝕜 (TangentSpace I x) := inferInstanceAs (Module 𝕜 E)
 instance : Inhabited (TangentSpace I x) := ⟨0⟩
+instance : ContinuousSMul 𝕜 (TangentSpace I x) := inferInstanceAs (ContinuousSMul 𝕜 E)
+-- the next instance derives from the previous one, but through an instance with priority 100
```

Comment:
Do you mean "this instance"? When reading "next", I'm thinking of an instance after `ContinuousConstSMul` one...

Reply:
`this instance` looks ambiguous to me, between the instance just before and the instance just after. `the following instance`, maybe?